seesaw/externalprocess.py

```diff
@@ -221,7 +225,10 @@ def fail_item(self, item):
         # Don't allow the item to fail until the external process completes
         if item["ExternalProcess.running"]:
             return
-        Task.fail_item(self, item)
+        if self.hard_exit == True:
```

Comment:
From what I see, `hard_exit` is already a boolean so it's redundant to compare it to another boolean